hack/update_crdb_versions.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
Would you mind moving this into `//hack/update_crdb_versions/main.go`? If we add more of these, they won't all be able to be in the main package since they've all got a main func. Just thinking about how this can serve as an example as we add more of these.